--- origsrc/ggobi-2.1.7/configure.ac	2007-09-28 08:23:44.000000000 -0500
+++ src/ggobi-2.1.7/configure.ac	2008-07-06 17:48:29.609375000 -0500
@@ -144,15 +144,10 @@
 # and to support DLLs on Windows. Also build libltdl for loading plugins.
 AC_DISABLE_STATIC
 AC_LIBTOOL_WIN32_DLL
-AC_LIBLTDL_CONVENIENCE
+AC_LIB_LTDL
 AC_LIBTOOL_DLOPEN
 AC_PROG_LIBTOOL
 
-# Configure libltdl build
-AC_SUBST(INCLTDL)
-AC_SUBST(LIBLTDL)
-AC_CONFIG_SUBDIRS(libltdl)
-
 # Configure for building against local libintl
 AC_SUBST(INCINTL)
 
