--- origsrc/ggobi-2.1.7/Makefile.am	2007-05-25 06:12:16.000000000 -0500
+++ src/ggobi-2.1.7/Makefile.am	2008-07-06 16:33:09.093750000 -0500
@@ -83,7 +83,7 @@
 
 	for i in $(PLUGIN_DIRS) ; do \
 	  echo "" >> $@ ; \
-	  cat plugins/$$i/plugin.xml >> $@ ; \
+	  cat $(top_srcdir)/plugins/$$i/plugin.xml >> $@ ; \
 	done
 	echo "</plugins>" >> $@ 
 	echo "</ggobirc>" >> $@
--- origsrc/ggobi-2.1.7/configure.ac	2007-09-28 08:23:44.000000000 -0500
+++ src/ggobi-2.1.7/configure.ac	2008-07-06 17:48:29.609375000 -0500
@@ -42,7 +42,7 @@
 AC_DEFINE_UNQUOTED([GGOBI_LOCALEDIR], "$GGOBI_LOCALEDIR", [Location of locale files])
 
 # gettext needs MKINSTALLDIRS, but for some reason isn't getting it
-MKINSTALLDIRS="\$(top_builddir)/$ac_aux_dir/mkinstalldirs"
+MKINSTALLDIRS="\$(top_srcdir)/config/mkinstalldirs"
 AC_SUBST(MKINSTALLDIRS)
 
 MAJOR_VERSION=ggobi_major_version
@@ -144,15 +144,10 @@
 # and to support DLLs on Windows. Also build libltdl for loading plugins.
 AC_DISABLE_STATIC
 AC_LIBTOOL_WIN32_DLL
-AC_LIBLTDL_CONVENIENCE
+AC_LIB_LTDL
 AC_LIBTOOL_DLOPEN
 AC_PROG_LIBTOOL
 
-# Configure libltdl build
-AC_SUBST(INCLTDL)
-AC_SUBST(LIBLTDL)
-AC_CONFIG_SUBDIRS(libltdl)
-
 # Configure for building against local libintl
 AC_SUBST(INCINTL)
 
--- origsrc/ggobi-2.1.7/ggobi.pc.in	2007-05-25 06:12:16.000000000 -0500
+++ src/ggobi-2.1.7/ggobi.pc.in	2008-07-06 16:33:09.250000000 -0500
@@ -2,14 +2,14 @@
 #
 prefix=@abs_top_srcdir@
 exec_prefix=${prefix}/src
-libdir=${prefix}/src
+libdir=@abs_top_builddir@/src
 includedir=${prefix}/src
 
 Name: libggobi
 Version: @PACKAGE_VERSION@
 Description: GGobi library for dynamic graphics.
 Requires: gtk+-2.0, libxml-2.0
-Libs: -L${libdir} -lggobi
-Cflags: -I${includedir} -I${prefix}/libltdl
+Libs: ${libdir}/libggobi.la
+Cflags: -I${includedir}
 
 Home: ${prefix}
--- origsrc/ggobi-2.1.7/plugins/Makefile.plugin	2007-05-25 06:11:17.000000000 -0500
+++ src/ggobi-2.1.7/plugins/Makefile.plugin	2008-07-06 16:33:09.296875000 -0500
@@ -6,7 +6,7 @@
 plugin_LTLIBRARIES        = plugin.la
 
 # Most plugins will only require GGobi and its dependencies (gtk, libxml2)
-plugin_la_LDFLAGS  = @GGOBI_LIBS@ -module
+plugin_la_LDFLAGS  = @GGOBI_LIBS@ -module -avoid-version -no-undefined -export-dynamic
 plugin_la_CFLAGS   = @GGOBI_CFLAGS@ @SRC_DEBUG@
 
 # Every plugin has an XML description and some docs
--- origsrc/ggobi-2.1.7/src/Makefile.am	2007-09-18 13:11:27.000000000 -0500
+++ src/ggobi-2.1.7/src/Makefile.am	2008-07-06 16:33:09.328125000 -0500
@@ -1,10 +1,11 @@
+
 lib_LTLIBRARIES     = libggobi.la
 libggobi_la_GOBS = data.gob renderer.gob renderer-factory.gob renderer-cairo.gob
 libggobi_la_GOBSOURCES = $(libggobi_la_GOBS:%.gob=ggobi-%.c)
 libggobi_la_GOBHEADERS = $(libggobi_la_GOBS:%.gob=ggobi-%.h)
 libggobi_la_GOBOBJECTS = $(libggobi_la_GOBS:%.gob=ggobi-%.lo)
 libggobi_la_GOBSTAMPS = $(libggobi_la_GOBS:%.gob=.%.gob.stamp)
-libggobi_la_LDFLAGS  = @GTK_LIBS@ @LIBXML_LIBS@
+libggobi_la_LDFLAGS  = @GTK_LIBS@ @LIBXML_LIBS@ -no-undefined
 libggobi_la_BUILT = array.c ash1d.c barchart.c barchartClass.c \
   barchart_ui.c brush.c brush_api.c brush_bins.c brush_init.c brush_link.c \
   brush_ui.c color.c color_ui.c cpanel.c display.c display_tree.c display_ui.c \
@@ -27,7 +28,7 @@
   write_csv.c write_xml.c wvis.c wvis_ui.c xlines.c xyplot.c \
   xyplot_ui.c $(libggobi_la_GOBSOURCES)
 libggobi_la_SOURCES = $(libggobi_la_BUILT) $(libggobi_la_GOBSTAMPS)
-libggobi_la_LIBADD = $(libggobi_la_GOBOBJECTS) @LIBLTDL@
+libggobi_la_LIBADD = $(libggobi_la_GOBOBJECTS) $(LIBLTDL)
 libggobi_la_DEPENDENCIES = $(libggobi_la_GOBOBJECTS)
 pkginclude_HEADERS = GGobiAPI.h ggobiClass.h tour1d_pp.h GGobiApp.h \
   noop-toggle.h tour2d_pp.h GGobiEvents.h parcoordsClass.h tour_pp.h \
@@ -52,7 +53,7 @@
 ggobi_SOURCES = ggobiMain.c
 ggobi_LDADD   = libggobi.la
 
-INCLUDES = @SRC_DEBUG@ @GTK_CFLAGS@ @LIBXML_CFLAGS@ @INCLTDL@
+INCLUDES = $(SRC_DEBUG) $(GTK_CFLAGS) $(LIBXML_CFLAGS) $(INCLTDL)
 
 EXTRA_DIST = $(libggobi_la_GOBS)
 
